### PR TITLE
Fix deprecation warnings ahead of Solidus `v3.0` upgrade

### PR DESCRIPTION
**Pull request by MadelineCollier**
- Fix firstname/lastname deprecation warnings (v3.0)
- Fix `Spree.t` deprecation warnings (v3.0)

### CODE REVIEW

File: app/views/spree/checkout/payment/_adyen_encrypted_cc.html.erb [hunk 1]
```diff
@@ -8,31 +8,31 @@
 <% param_prefix = "payment_source[#{payment_method.id}]" %>
 
 <div class="adyen-encrypted-cc field">
-  <%= label_tag nil, Spree.t(:name_on_card) %><span class="required">*</span><br />
-  <%= text_field_tag nil, "#{@order.billing_firstname} #{@order.billing_lastname}", { autocomplete: "off", size: 20, id: nil, "data-encrypted-name" => "holderName"} %>
+  <%= label_tag nil, I18n.t(:name_on_card) %><span class="required">*</span><br />
```
Comment (jarednorman) [inline app/views/spree/checkout/payment/_adyen_encrypted_cc.html.erb]: In views this should use the view helper:

```suggestion
  <%= label_tag nil, t(:name_on_card) %><span class="required">*</span><br />
```

**Review by jarednorman — CHANGES_REQUESTED** [review]

File: app/views/spree/checkout/payment/_adyen_encrypted_cc.html.erb (hunk 1 above)
Comment (MadelineCollier) [reply app/views/spree/checkout/payment/_adyen_encrypted_cc.html.erb]: Ok cool will fix ty 👍🏻

Comment (MadelineCollier) [reply app/views/spree/checkout/payment/_adyen_encrypted_cc.html.erb]: Fixed

Comment (benjaminwil) [reply app/views/spree/checkout/payment/_adyen_encrypted_cc.html.erb]: Yeah. `I18n#t` won't know about the implied scope of the translation key `.name_on_card` but `#t` will.

**Review by benjaminwil — APPROVED** [review]

**Review by jarednorman — APPROVED** [review]

**Pull request merged**